apps/melding-form/src/app/(map)/locatie/kies/_components/_utils/updateWfsLayer.test.ts

```diff
@@ -0,0 +1,74 @@
+import { buildWfsLayer } from './buildWfsLayer'
+import { updateWfsLayer } from './updateWfsLayer'
+
+vi.mock('./buildWfsLayer', () => ({
+  buildWfsLayer: vi.fn(),
+}))
+
+describe('updateWfsLayer', () => {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  let mapInstance: any
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  let assetLayerRef: { current: any }
+
+  beforeEach(() => {
+    assetLayerRef = { current: null }
+    mapInstance = {
+      getZoom: vi.fn(),
+    }
+    vi.clearAllMocks()
+  })
```

Comment:
This is the third different way to do this 😁 Let's pick the best one and use that everywhere.